Lesson13/dicle/script.js

```diff
@@ -0,0 +1,181 @@
+  parseInt('50'),
+  parseInt('2.5'),
+  parseInt('2,5'),
+  parseInt('Batikan')
+);
+
+// 11. Generate a random integer between 2 and 10 and log it.
+console.log(Math.random()); //0.8763250588907179
+const randomFrom2To10 = Math.random() * 8 + 2;
+console.log('Ex. 11', Math.floor(randomFrom2To10));
+
+// 12. Round the number 3.7 down using Math.floor and 3.2 up using Math.ceil, log both.
+const floorNumber = 3.7;
+const ceilNumber = 3.2;
+console.log('Ex. 12', Math.floor(floorNumber), Math.ceil(ceilNumber));
+
+// 13. Declare a boolean variable isStudent = true. Log it.
+const isStudent = true;
+console.log('Ex. 13', isStudent);
+
+// 14. Initialize counter = 0, then increment it by 1 using counter++ and log it.
+let counter = 0;
+counter++;
+console.log('Ex. 14', counter);
+
+// 15. Initialize points = 10, add 5 to it using points += 5, then log points.
+let points = 10;
+const plusPoint = (points += 5);
+console.log('Ex. 15', points, plusPoint);
+
+// 16. Declare name="Alice", age=30, city="Paris". Log "Alice (30) lives in Paris" using template literals.
+const name = 'Alice';
+const age = '30';
+const city = 'Paris';
+console.log('Ex. 16', `${name} ${age} lives in ${city}`); //Can't do the paranthesis?
+
+// 17. Declare variables x=5, y=10, z=15. Log their total sum.
+const xNumber = 5;
+const yNumber = 10;
+const zNumber = 15;
+const totalSum = xNumber + yNumber + zNumber;
+console.log('Ex. 17', totalSum);
+
+// 18. Declare dividend=10 and divisor=3. Log the quotient (divisionResult) and difference (differenceResult).
+let dividend = 10;
+let divisor = 3;
+
+let divisionResult = dividend / divisor;
+let differenceResult = dividend - divisor;
+console.log('Ex. 18', divisionResult, differenceResult);
+
+// 19. Declare firstName and lastName. Create fullName by concatenating them with a space and log it.
+const firstName = 'Dicle';
+const lastName = 'Arpak';
+const fullName = firstName + ' ' + lastName;
+console.log('Ex. 19', fullName);
+
+// 20. Declare firstFactor=7 and secondFactor=2. Log the product.
+
+const firstFactor = 7;
+const secondFactor = 2;
+const product = firstFactor * secondFactor;
+console.log('Ex. 20', product);
+
+// 21. Log the value of Math.PI.
+console.log('Ex. 21', Math.PI);
+
+// 22. Declare counter=0. Increment it using three different methods (e.g., counter++, counter+=1, counter=counter+1) and log the result each time.
+let counter2 = 0;
+let counter3 = counter2++;
+let counter4 = (counter2 += 1);
+let counter5 = counter2 + 1;
+console.log('Ex. 22', counter2);
+console.log(counter3);
+console.log(counter4);
+console.log(counter5);
+
+// 23. Declare initialTemperature=20. Increase it by 5 and log the result.
+let initialTemperature = 20;
+let increasedTemperature = initialTemperature + 5;
+console.log('Ex. 23', increasedTemperature);
+
+// 24. Declare numberEx9=6. Increment it using the prefix ++ operator and log both the variable and the incremented value.
+let numberEx9 = 6;
+let incrementedNumber = ++numberEx9;
```

Comment:
You don't need to declare a new variable here. But if you do, use `const` instead of `let` since the value never changes.
```suggestion
const incrementedNumber = ++numberEx9;
```